gnomad_qc/v3/load_data/load_v2_qc_sites.py

```diff
@@ -1,30 +1,25 @@
 from gnomad_hail import *
 from gnomad_qc.v2.resources.sample_qc import qc_mt_path
+from gnomad_qc.v3.resources import gnomad_v2_qc_sites
 import argparse
 
 
-def gnomad_v2_qc_sites(overwrite: bool):
+def main(args):
+
+    hl.init(log='/load_resources.hail.log', default_reference='GRCh38')
+    hl.get_reference('GRCh37').add_liftover('gs://hail-common/references/grch37_to_grch38.over.chain.gz', hl.get_reference('GRCh38'))
+
     v2_qc_sites = hl.read_matrix_table(qc_mt_path('joint', ld_pruned=True)).rows().select()
     v2_qc_sites = v2_qc_sites.key_by()
     v2_qc_sites = v2_qc_sites.transmute(
         locus=hl.liftover(v2_qc_sites.locus, 'GRCh38'),
         locus37=v2_qc_sites.locus
     )
     v2_qc_sites = v2_qc_sites.filter(v2_qc_sites.locus.contig == 'chr' + v2_qc_sites.locus37.contig).key_by('locus', 'alleles')
```

Comment:
For my knowledge, why do you only filter to where the contig matches and not the position? Isn't it possible the position does not lift over?

Reply:
Well, the position will change between the builds. I'm removing the few loci where the contig changes as I don't trust these quite as much -- but to be entirely fair this is more of a gut feeling than a strict evaluation. What I can say is that those that move contig don't lift-under well afterwards so it made me suspicious; it's also a very small number of loci.